tests/integration/netapi/rest_cherrypy/test_app.py

```diff
@@ -190,7 +193,7 @@ def test_accepts_arg_kwarg_keys(self):
                 'Accept': 'application/json',
             }
         )
-        resp = json.loads(response.body[0])
+        resp = json.loads(salt.utils.to_str(response.body[0]))
```

Comment:
Wait! Wait!
Sorry @terminalmage, I was being dumb.
The content type is  'application/json', so it must be UTF-8, and in this case, pass it explicitly.